staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -490,6 +490,14 @@ type KubeletConfiguration struct {
 	// Default: "/etc/resolv.conf"
 	// +optional
 	ResolverConfig string `json:"resolvConf,omitempty"`
+	// TODO(mtaufen): It is increasingly looking like nobody actually uses the
```

Comment:
You can remove this comment :). If we ever decide to remove it, it would best be done as a `v2` API in the distant future.